test/unit/script/plugin_webpack_directories_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'minitest/autorun'
+require 'json'
+
+class PluginWebpackDirectoriesTest < Minitest::Test
+  def setup
+    @root_dir = File.join('../../../..', __FILE__)
```

Comment:
nitpicking, but the whole value of using file.join was to not include / or \ so ['..','..','..'....]
also, shouldnt it be 3 times vs 4 ?

if you want to be sure you can probably assert that @root_dir exists is rails root too